pkg/backend.go

```diff
@@ -0,0 +1,75 @@
+package pkg
```

Comment:
please remove this file - use cmd/compile.go

Reply:
Maybe it's better to create subpackage like `backend` or something like this?
Because as far as I see ideas of cmd/compile.go and backeng.go are quite different - `compile.go` is used as a part of binary for console usage and `backend.go` is for usage inside of service(c), to be able to compile SEAL rules stored in database, as an example

Reply:
the open-source repo should remain minimalistic and won't be implementation specific things like accessing databases.

The backends are in pkg/compiler/* - see rego.  that compiler backend compiles seal to rego and is slated to build rego for general authz.